src/rtctools/optimization/multi_seed_mixin.py

```diff
@@ -0,0 +1,78 @@
+from pathlib import Path
+from typing import Dict, Iterable, Optional
+
+from rtctools.optimization.optimization_problem import OptimizationProblem
+from rtctools.optimization.timeseries import Timeseries
+
+
+class MultiSeedMixin(OptimizationProblem):
+    """
+    Enables a workflow to solve an optimization problem by trying multiple seeds.
+    """
+
+    def __init__(self, **kwargs):
+        self.__seed_file: Path = None
+        super().__init__(**kwargs)
+
+    def use_seed_file(self):
+        """Return True if the seed file is used for the current run.
+
+        By default, this is True.
+        Overwrite this method when in some cases the seed is not based on the seed file,
+        e.g. in case of goal programming, return False after the first priority,
+        or in case of homotopy, return False after the homotopy parameter is increased.
+        """
+        return True
+
+    def seed_file(self) -> Optional[Path]:
```

Comment:
What is this method used for?

Reply:
I have renamed the methods somewhat. This will now return the selected seed id (before the selected seed file).